### DISCUSSION TITLE
IServiceCollection does not contain a definition for 'AddHandlers'

### DISCUSSION THREAD

**Discussion by needsleep55** [ImmediatePlatform/Immediate.Handlers#67] [Q&A] [answered]
I am trying to register handlers in a project but I am getting the error mentioned in the title. I installed the package with NuGet and have a using statement for `Immediate.Handlers.Shared` in my GlobalUsings.cs file, but I'm not sure what else I'm missing in order to properly call the `AddHandlers` method.

**Comment by viceroypenguin** [comment, marked as answer]
The `AddHandlers()` extension method is defined in the root namespace of your project. If you have a `<RootNamespace/>` in your .csproj file, `using` that. Otherwise, `using` the name of your .csproj file (aka `Web.csproj` would be `using Web;`.

**Comment by needsleep55** [comment]
Great that did it thanks!
